### DISCUSSION TITLE
Windows Defender

### DISCUSSION THREAD

**Discussion by ScalableCodex** [FreeTubeApp/FreeTube#2516] [General]
Windows Defender wanted me to send the History.db file today in in the roaming folder. Anyone else get this? I assume Microsoft is just smoking something to think it's an issue.

It keeps asking over and over.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
im running a windows machine and this never happened to me

**Reply by ScalableCodex** [reply]
Yeah it's weird. It only did it after I updated to the newest one, and did it like 7 times until I rebooted, and it hasn't done it since.

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
That is indeed very weird. Nobody else reported such an issue so i think its just ur windows defender going crazy

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Im going to close this thread as it seems that windows defender is just doing random stuff that isnt related to FT.

**Comment by Nekothegamer** [comment]
how do you even have History.db, google takeout exported my history as a .csv

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
If u dont import a csv from google takeout and just start watching videos FT will create a DB file

**Reply by Nekothegamer** [reply]
yeah but i want to import my history from youtube, is there a way to convert .csv to .db

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
Importing history from CSV / JSON is not supported at this time unfortunately. There is an issue proposing this functionality (#1623).
